packages/frontend/src/pages/ImportModels.svelte

```diff
@@ -0,0 +1,147 @@
+import { Uri } from '@shared/src/uri/Uri';
+import Button from '../lib/button/Button.svelte';
+import ErrorMessage from '../lib/ErrorMessage.svelte';
+import NavPage from '../lib/NavPage.svelte';
+import type { LocalModelImportInfo } from '@shared/src/models/ILocalModelInfo';
+
+let modelsToImport: LocalModelImportInfo[] = [];
+let importError: string = '';
+let addModelError: string = '';
+
+let inProgress = false;
+
+$: importDisabled = modelsToImport.length === 0;
+
+async function addModelsToImport() {
+  addModelError = '';
+  try {
+    const models = await studioClient.openDialog({
+      title: 'Select models to import',
+      selectors: ['multiSelections', 'openFile'],
+      filters: [
+        {
+          name: 'GGUF files',
+          extensions: ['gguf'],
+        },
+      ],
+    });
+    if (!models) {
+      return;
+    }
+    const modelsInfo: LocalModelImportInfo[] = [];
+    models.forEach(model => {
+      model = Uri.revive(model);
+      const modelPath = model.path;
+      // we show an initial name that can be modified by the user.
+      const lastSlashIndex = modelPath.replace(/\\/g, '/').lastIndexOf('/') + 1;
+
+      if (!modelsToImport.find(i => i.path === modelPath)) {
+        modelsInfo.push({
+          path: modelPath,
+          name: modelPath.substring(lastSlashIndex).replace('.gguf', ''),
+        });
+      }
+    });
+    const invalidModelsToImport = await studioClient.checkInvalidModels(modelsInfo.map(m => m.path));
+    if (invalidModelsToImport.length > 0) {
+      addModelError = `Error: the following ${invalidModelsToImport.length === 1 ? 'model is' : 'models are'} already in the catalog and cannot be imported: `;
+      invalidModelsToImport.forEach(path => {
+        addModelError += `${path} ; `;
+      });
+      modelsToImport = [...modelsToImport, ...modelsInfo.filter(m => !invalidModelsToImport.includes(m.path))];
+    } else {
+      modelsToImport = [...modelsToImport, ...modelsInfo];
+    }
+  } catch (e) {
+    addModelError = `Error while adding models: ${String(e)}`;
+  }
+}
+
+function onModelNameInput(event: Event, index: number) {
+  const target = event.currentTarget as HTMLInputElement;
+  modelsToImport[index].name = target.value;
+  modelsToImport = modelsToImport;
+}
+
+function deleteModelToImport(index: number) {
+  modelsToImport = modelsToImport.filter((_, i) => i !== index);
+}
+
+async function importModels() {
+  importError = '';
+  inProgress = true;
+
+  try {
+    await studioClient.importModels(modelsToImport);
+  } catch (e) {
+    importError = `Error while importing models: ${String(e)}\n`;
+  }
+
+  inProgress = false;
+  if (importError === '') {
+    router.goto(`/models`);
+  }
+}
+</script>
+
+<NavPage lastPage="{{ name: 'Models', path: '/models' }}" title="Import Models" searchEnabled="{false}">
+  <svelte:fragment slot="content">
+    <div class="p-5 min-w-full h-fit">
+      <div class="bg-charcoal-800 rounded-lg px-6 py-4 space-y-2">
```

Comment:
```suggestion
      <div class="bg-charcoal-800 rounded-lg px-6 py-6 space-y-2">
              <span class="block mb-2 text-sm font-medium text-gray-400">Models to import:</span>
```

To match the design of the import containers

Here is the example
![image](https://github.com/containers/podman-desktop-extension-ai-lab/assets/42176370/06fa97a8-53ed-49e7-9fd9-4641e04d7a14)

with the changes you get
![image](https://github.com/containers/podman-desktop-extension-ai-lab/assets/42176370/529ec43d-1294-468a-ad8d-dd7f78b9cbb3)